### PR TITLE
Tidying up the codebase

### PR DESCRIPTION
**Pull request by mcsaeid**

### CODE REVIEW

File: pkg/reddit/fetch.go [hunk 1]
```diff
@@ -304,10 +304,10 @@ func getPost(postUrl string, root map[string]interface{}) (fetchResult interface
 				Duration:      int64(duration),
 				Type:          FetchResultMediaTypeVideo,
 			}, nil
-		case "rich:video": // files hosted other than reddit; This bot currently supports gfycat.com
+		case "rich:video": // files hosted other than reddit; This bot currently supports Gfycat.com
 			if urlObject, domainExists := root["domain"]; domainExists {
 				switch urlObject.(string) {
-				case "gfycat.com": // just act like gif
+				case "Gfycat.com": // just act like gif
```
Comment (HirbodBehnam) [inline pkg/reddit/fetch.go:310]: I think this should not be like this. The domain must be lowercase

File: pkg/reddit/fetch.go [hunk 2]
```diff
@@ -246,10 +246,10 @@ func getPost(postUrl string, root map[string]interface{}) (fetchResult interface
 				if strings.HasPrefix(root["url"].(string), "https://i.imgur.com") { // Example: https://www.reddit.com/r/dankmemes/comments/gag117/you_daughter_of_a_bitch_im_in/
 					gifDownloadUrl := root["url"].(string)
 					lastSlash := strings.LastIndex(gifDownloadUrl, "/")
-					gifDownloadUrl = gifDownloadUrl[:lastSlash+1] + "download" + gifDownloadUrl[lastSlash:]
+					gifDownloadUrl = gifDownloadUrl[:lastSlash+1] + "Download" + gifDownloadUrl[lastSlash:]
```
Comment (HirbodBehnam) [inline pkg/reddit/fetch.go:249]: I think this one must be in lowercase as well

**Comment by HirbodBehnam** [comment]
Very nice. Needs some small changes which I'll do myself.

Much obliged!

File: pkg/reddit/fetch.go (hunk 1 above)
Comment (mcsaeid) [reply pkg/reddit/fetch.go:310]: It should work either way, but I agree. I did a find-and-replace, and that’s how that got there. (: I’ll change this back on my side.

File: pkg/reddit/fetch.go (hunk 2 above)
Comment (mcsaeid) [reply pkg/reddit/fetch.go:249]: Sure.

**Comment by mcsaeid** [comment]
You are *most* welcome!

**Pull request merged**